python/pylibcudf/pylibcudf/libcudf/nvtext/minhash.pxd

```diff
@@ -34,6 +42,14 @@ cdef extern from "nvtext/minhash.hpp" namespace "nvtext" nogil:
         const size_type width,
     ) except +
 
+    cdef unique_ptr[column] minhash64_permuted(
+        const column_view &strings,
+        const uint64_t seed,
+        const column_view &a,
+        const column_view &b,
```

Comment:
This is interesting. I thought the signatures has to match.
it's `device_span` in `nvtext/minhash.hpp`, but defined as `column_view`.
is implicit conversion happening here?

Reply:
Yes. I've seen this used elsewhere as well.

Reply:
`column_view` is implicitly convertible to `device_span`. This forward-declaration doesn't have to "match" the C++ in any way, it's purely for type checking in Cython. When the C++ is emitted we will have `minhash64_permuted(..., a_column_view, ...)` and the normal C++ implicit conversion kicks in.